languages/de-DE/ControlPanel.multids

```diff
@@ -25,6 +25,12 @@ Basics/Tiddlers/Prompt: Anzahl Tiddler
 Basics/Title/Prompt: Titel dieses ~TiddlyWikis
 Basics/Username/Prompt: Benutzersignatur zum Editieren
 Basics/Version/Prompt: ~TiddlyWiki Version
+Client/Caption: Client-Optimierungen
+Client/Hint: Hier können Sie das Erscheinungsbild Ihres Kunden anpassen.
+Client/Tiddloid/Caption: Tiddloid
+Client/Tiddloid/Hint: Diese Einstellungen gelten für den Tiddloid-Client.
+Client/Tiddloid/ApplyTheme/Description: Wenden Sie die Themenfarbe auf die Systemleisten an
+Client/Tiddloid/HideToolbar/Description: Ausblenden Symbolleiste auf der Ladevorgang abgeschlossen
```

Comment:
IMO this isn't proper German. ... And I'm not sure what is meant. Even the English text isn't clear to me

Reply:
Okay...I'm not German speaker XD I just uses google translate

Reply:
The first checkbox is to hide the appbar of Tiddloid if the file is completely loaded, and the second is to let appbar and systembar be the same color as the background of the page, as shown in the picture.